about.html

```diff
@@ -0,0 +1,96 @@
+            <a href="./index.html"><img class="header-logo" src="./image/headerLogo.png" alt="logo"></a>
+            <div class="desktop-menu d-flex">
+                <div class="menu-desktop d-flex gap-5 p-4">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#main-program">Program</a>
+                    <a href="#main-program">Join</a>
+                    <a href="#partner">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="./about.html">Contact</a>
+                </div>
+            </div>
+        </div>
+        <div class="top container-fluid pt-4 p-4">
+            <header class="header fixed-top m-4">
+                <div class="bar" id="bar1"></div>
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                <p class="border border-3 border-white mt-5">Carbon capture, utilisation, and storage, or CCUS, is an emerging technology for reducing carbon emissions produced from energy production, heavy industries, and transportation. The captured CO2 can be used as a valuable resource or permanently stored deep underground in geological formations, providing the foundation for carbon removal from the atmosphere. </p>
+                <div class="contact text-center">
+                    <h5>Please contact us via email for any further question</h5>
+                    <h5>About CC Global Summit 2022!</h5>
+                    <h5 id="email" class="email text-dark fs-6"><u>ccsummit2022@ccglobal.org</u></h5>
+                </div>
+            </main>
+        </div>
+    </div>
+
+    <div class="partner bg-white">
+        <h5 class="partners_heading text-center text-dark pt-4">Creative Commons</h5>
+        <h5 class="partners_heading text-center text-dark">Global Summit 2022 logo</h5>
+        <h3 class="text-danger text-center">____</h3>
+        <p class="text-center p-4">The logo of CC Global summit 2022 decided through the logo competion from 8 June to 7 July</p>
+        <div class="w-100 text-center">
+            <img class="p-4" src="./image/logo.png" alt="CC Summit 2022 logo" id="logo-aboutme">
+        </div>
+        <h5 class="partners_heading text-center text-dark">See the Past Global summits</h5>
+        <h3 class="text-danger text-center">____</h3>
+        <p class="text-center p-4">Take a look at last two CC Global Summits, which took place in Buenos Aires and Warsaw.</p>
+        <img class="p-4 w-100" src="./image/about_2011.png" alt="2011 CC summit icon">
+        <img class="p-4 w-100" src="./image/about_2013.png" alt="2013 CC summit icon">
```

Comment:
- [ ] Please ensure that the images are aligned in a row in the past events section of your project's desktop version. Right now they are in a vertical column as you can see below
> Yours
![image](https://user-images.githubusercontent.com/90556221/183047106-2b298c17-2fd4-41e8-b13d-e20627bbc986.png)
> Behance
![image](https://user-images.githubusercontent.com/90556221/183047167-bb6ea2ce-7240-4658-ab49-59c8e3565747.png)

Reply:
Resolved the issue.